charts/argo-workflows/Chart.yaml

```diff
@@ -10,5 +10,5 @@ maintainers:
 version: 0.0.13
```

Comment:
```suggestion
version: 0.0.14
```

Reply:
Making sure we're up to date with everything argo.

Reply:
you know you can commit, approve, merge all of them as well, if need be.